<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="entrateBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Entrate</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEN0">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEN1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEN2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEN3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEN4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEN5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEN6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEN7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="usciteBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Uscite</string>
    </property>
    <widget class="QPushButton" name="pb_U0">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{background-color: rgb(38, 162, 105);}
QPushButton{background-color: rgb(165, 29, 45);}</string>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_U1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{background-color: rgb(38, 162, 105);}
QPushButton{background-color: rgb(165, 29, 45);}</string>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_U2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{background-color: rgb(38, 162, 105);}
QPushButton{background-color: rgb(165, 29, 45);}</string>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_U3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{background-color: rgb(38, 162, 105);}
QPushButton{background-color: rgb(165, 29, 45);}</string>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_U4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{background-color: rgb(38, 162, 105);}
QPushButton{background-color: rgb(165, 29, 45);}</string>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_U5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{background-color: rgb(38, 162, 105);}
QPushButton{background-color: rgb(165, 29, 45);}</string>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>44</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="statusConnectionLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection status</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>213</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="inputMask">
        <string>000.000.000.000</string>
       </property>
       <property name="text">
        <string>192.168.2.200</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
